### ISSUE TITLE
Using gulp-uncss with postcss

### ISSUE THREAD

**Issue by codedbypaul** [ben-eb/gulp-uncss#39]
I'm having an issue getting gulp-uncss playing nicely with postcss. I was originally using gulp-sass which worked fine.

But using postcss I get this error:
```
/Applications/MAMP/htdocs/wpn-chameleon/addition/node_modules/gulp-uncss/node_modules/uncss/node_modules/bluebird/js/main/async.js:43
        fn = function () { throw arg; };
                                 ^
Error: UnCSS: no stylesheets found
    at process (/Applications/MAMP/htdocs/wpn-chameleon/addition/node_modules/gulp-uncss/node_modules/uncss/src/uncss.js:145:15)
```

My gulp css task looks like this:
```
module.exports = function(gulp, plugins, config) {
  return function() {
    return gulp.src(config.development.css.src + '{style,style.ie}.css')
      .pipe(plugins.postcss(processors))
      .pipe(plugins.uncss({
        html: ['./build/development/*.html'],
        ignore: [/(ie7|ie8)/, /\.active/g]
      }))
      .pipe(gulp.dest(config.development.css.dest))
  }
}
```

Anything obvious as to why it's not working?

**Comment by ben-eb** [comment]
Works fine for me:

```js
var postcss = require('gulp-postcss');
var uncss = require('gulp-uncss');
var gulp = require('gulp');

gulp.task('default', function () {
  return gulp.src('./fixture.css')
    .pipe(postcss([ require('postcss-size') ]))
    .pipe(uncss({html: ['./index.html']}))
    .pipe(gulp.dest('out'));
});
```

fixture.css:

```css
h1, h2 {
  size: 50px;
}
```

index.html:

```html
<html><body><h1>hello</h1></body></html>
```

output:

```css
h1 {
  width: 50px;
  height: 50px;
}
```

**Comment by codedbypaul** [comment]
Ok. After changing the source to a single css file it works.

Seems it doesn't like:
```
return gulp.src(config.development.css.src + {style,style.ie}.css
```

**Comment by ben-eb** [comment]
OK, no problem. I've added a check that will simply pass through the CSS when the file is empty; as uncss'ing an empty string should return an empty string and not a crash. :smile:

It's available since 1.0.4.

**Comment by codedbypaul** [comment]
Cool. :)
